platform.md

```diff
@@ -85,6 +85,18 @@ The platform MUST ensure that:
 - The image config's `Label` field has the label `io.buildpacks.stack.id` set to the stack ID.
 - The image config's `Label` field has the label `io.buildpacks.stack.mixins` set to a JSON array containing mixin names for each mixin applied to the image.
 
+The platform SHOULD ensure that:
+
+- The image config's `Label` field has the label `io.buildpacks.stack.maintainer` set to the name of the stack maintainer.
+- The image config's `Label` field has the label `io.buildpacks.stack.homepage` set to the homepage of the stack.
+- The image config's `Label` field has the label `io.buildpacks.stack.distro.name` set to the name of the stack's OS distro.
+- The image config's `Label` field has the label `io.buildpacks.stack.distro.version` set to the version of the stack's OS distro.
+- The image config's `Label` field has the label `io.buildpacks.stack.version` set to the release number of the stack.
```

Comment:
```suggestion
- The image config's `Label` field has the label `io.buildpacks.stack.version` set to the version of the stack.
```
My suggested wording is a bit tautological but I think `release number` is too precriptive. I think folks will understand the above wording but we could also go with something wordier like
```
has the label `io.buildpacks.stack.version` set to a value that uniquely identifies a released version of the stack
```

Reply:
Also, perhaps this should have been a discussion for the RFC, but do we actually want to enforce lock step versioning between build and run images for all stacks? Given that build images have many more dependencies than run images, this would frequently require run-image releases where nothing has changed besides this one label.

Reply:
Looking at <https://github.com/buildpacks/spec/blob/main/platform.md#stacks>, it doesn't look like we define anything wrt versioning. If I update a build only dependency but it doesn't have a runtime update, do I need to rev build/run images to match then?

Reply:
@hone 
> If I update a build only dependency but it doesn't have a runtime update, do I need to rev build/run images to match then?

If we must express versions using a single `io.buildpacks.stack.version` label, I think it would be impractical to rev them independently. The name implies that the version describes the entire stack, rather than a particular image.

Reply:
@ekcasey right, so versions transcend the images specifically?

Reply:
@hone As it is written now they do, I am questioning whether we actually want that.

Reply:
Wherever we end up, it's probably worth calling it out explicitly in this section.

Reply:
Is the problem here perhaps that these labels should only contain information about the run image part of a stack implementation?  I can see that there is some utility to knowing about the image that was used during the build, but it feels like the big win here is looking at the run image under this specific application.